### PR TITLE
[Avatar Addition] Ori'Vali

### PR DESCRIPTION
**Pull request by Polaris-Avali**
Ori'vali Base: https://mikedabird.itch.io/orivali

### CODE REVIEW

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "./data_schema.json",
-  "JSON Version": "2.19.0",
-  "JSON Date": "2024-06-07",
+  "JSON Version": "2.19.1",
```
Comment (Happyrobot33) [inline Packages/com.mattshark.openflight/Runtime/data.json:3]: Version not incremented properly, increment the 19 tp 20, not the patch field like you did

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "FunnelVision",
     "PhoenixFallen",
     "-Dark-Spirit-"
```
Comment (Happyrobot33) [inline Packages/com.mattshark.openflight/Runtime/data.json:30]: Missing a comma here, making the list invalid

**Review by Happyrobot33 — CHANGES_REQUESTED** [review]

**Comment by Polaris-Avali** [comment]
Fixed the syntax errors as noted, lemme know if I missed anything else

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 3]
```diff
@@ -27,7 +27,8 @@
     "Mіcca",
     "FunnelVision",
     "PhoenixFallen",
-    "-Dark-Spirit-"
+    "-Dark-Spirit-",
+    "Mike Da Bird"
```
Comment (Happyrobot33) [inline Packages/com.mattshark.openflight/Runtime/data.json:31]: Just to ask by the way, this field is for the user that contributed the has to the system. Are you sure you want mike to be here, not you?

**Pull request merged**